### DISCUSSION TITLE
Care to share your configs? 🚀

### DISCUSSION THREAD

**Discussion by aur3l14no** [vscode-neovim/vscode-neovim#937] [General]
I'm previously a VSCode + vim (only for light editting with https://github.com/amix/vimrc and no personal config) + Emacs user.

Since neovim shows the potential of being a universal vim-style editing backend, I decide to invest some time and configure it.

I want to start by copying others' configurations. However, I'm concerened that the general neovim configs might not be compatible with vscode-neovim, and I'm even unable to tell the compatibility beforehand (cuz most neovim users don't care about VSCode and will not mention it in their repos). Yes I can just try it myself, but a bit help from others in the beginning makes things easier. 😉 Moreoever, I'd like to use firenvim which might share the similar problem.

Therefore, I'd really appreciate it if someone here can share some experience on:

1. What is your configuration?
2. What do I need to do to create a vscode-friendly and firenvim-friendly (I know it's a bit off topic, sorry in advance) configuration? Any other suggestions besides those mentioned in README already?

**Comment by theol0403** [comment]
To use same config for nvim/vscode/firenvim, use conditionals. For plugins, don't put the plugin definitions in a conditional, conditionally load the plugin, as shown in readme.

For compatibility, follow readme about what types of plugins aren't supported. We should improve the wiki for supported plugins.

Here is my lua config:
https://github.com/theol0403/dotfiles/tree/master/.config/nvim/vimrc/moon

**Comment by theol0403** [comment]
The best configuration, by far, is currently https://www.lazyvim.org/plugins/extras/vscode

**Reply by ian-h-chamberlain** [reply]
Looks like this might have moved (https://www.lazyvim.org/extras/vscode) ?
